h2o-algos/src/main/java/hex/kmeans/KMeans.java

```diff
@@ -357,8 +357,8 @@ public void computeImpl() {
         vecs2[vecs2.length-1].remove();
 
         // Create metrics by scoring on training set otherwise scores are based on last Lloyd iteration
-        model.score(_train).delete();
-        model._output._training_metrics = ModelMetrics.getFromDKV(model,_train);
+        model.score(_parms.train()).delete();
```

Comment:
I do not believe this is correct, scoring needs to be done on the adapted dataset, not the original dataset with extra columns

Reply:
This comment is incorrect ^^^ please ignore. Model#score calls adaptTestToTrain - which makes this ok